src/components/SearchFilter/SearchFilter.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from "react";
+import { useForm } from "react-hook-form";
+import { useHistory } from "react-router-dom";
+import { Select } from "../Select/Select";
+import { DataList } from "../DataList/DataList";
+import { SERVICES, DATALIST, PETS } from "../../constants/SearchOptions";
+import AnnouncementsContext from "../../context/announcements/announcements-context";
+import styles from "./SearchFilter.module.scss";
+
+export const SearchFilter = () => {
+  const history = useHistory();
+  const announcementsCtx = useContext(AnnouncementsContext);
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const onSubmit = (data) => {
+    announcementsCtx.filter(data);
+    if (window.location.pathname === "/") {
+      history.push({
+        pathname: "/announcements",
+      });
+    }
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)} className={styles.form}>
```

Comment:
So it is not a `SearchFilter` but more like `SearchForm`. Is it gonna be reusable or not? If no, then what is the point of `if (window.location.pathname === "/")` statement?

Reply:
It is used on main page and page with list of announcements